geometry/build.gradle

```diff
@@ -41,9 +40,6 @@ java {
 }
 }
 
-compileJava.dependsOn 'spotlessApply'
```

Comment:
can we add this back so formatting happens?

Reply:
its on line 68